Gemfile.lock

```diff
@@ -230,5 +234,8 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console
 
+RUBY VERSION
+   ruby 2.3.1p112
+
```

Comment:
I would get rid of it, but it was added in https://github.com/alphagov/com-prototype/pull/2 in https://github.com/alphagov/com-prototype/pull/2/commits/6e0691a054991bad257db1794f86cd15620787a8. 
Not a big deal, though. It might be easier for the first sprints to rely only on `.ruby-version`